openupgrade_scripts/scripts/payment/16.0.2.0/upgrade_analysis.txt

```diff
@@ -0,0 +1,179 @@
+---Models in module 'payment'---
+obsolete model payment.acquirer
+obsolete model payment.acquirer.onboarding.wizard [transient]
+model payment.refund.wizard (moved to account_payment) [transient]
+new model payment.provider
+new model payment.provider.onboarding.wizard [transient]
```

Comment:
FYI those two changes are model rename (`acquirer` -> `provider`)
will probably simplify the upgrade analysis ;)

(Applies to view & other records xml ids as well, in `payment(_*)` modules, we replaced all occurrences of `acquirer`)